### DISCUSSION TITLE
php 容器安装的扩展不在 /php/extensions 下 在/usr/local/lib/php/extensions 在

### DISCUSSION THREAD

**Discussion by hsjgit** [1Panel-dev/1Panel#5566] [Q&A]
### 1Panel 版本

v1.10.10-lts

### 问题描述

php 容器安装的扩展不在 /php/extensions 下 在/usr/local/lib/php/extensions 在

### 重现步骤

安装一个php环境， 进入容器 ls /usr/local/lib/php/extensions

### 期待的正确结果

挂在出正确的目录

**Comment by wanghe-fit2cloud** [comment]
挂载 /php/extensions 目录，用于手动安装扩展，脚本默认指定使用此目录。

**Reply by hsjgit** [reply]
我构建了php的基础镜像之后，使用的时候发现有的扩展用不到，我就想卸掉。然后我就只能带容器中去把那行扩展注释掉。但是容器down之后重新up起来被办法持久化，我就希望能把默认的也挂在出来。但是我修改了挂在目录之后，默认的扩展目录挂载出来是空目录。

**Reply by hsjgit** [reply]
/usr/local/etc/php/conf.d 我是想把这个目录挂载出来。
